src/disco/gui/fd_gui.c

```diff
@@ -1312,6 +1312,17 @@ fd_gui_handle_rooted_slot( fd_gui_t * gui,
     slot->level = FD_GUI_SLOT_LEVEL_ROOTED;
     fd_gui_printf_slot( gui, parent_slot );
     fd_http_server_ws_broadcast( gui->http );
+
+    /* Broadcast new skip rate if one of our slots got rooted. */
```

Comment:
I think rooted is the wrong place for this, should be if a slot is "completed" or flips to skipped (or back from skipped).